contracts/modules/TransferManager/ManualApprovalTransferManager.sol

```diff
@@ -19,20 +19,19 @@ contract ManualApprovalTransferManager is ITransferManager {
 
     //Manual approval is an allowance (that has been approved) with an expiry time
     struct ManualApproval {
+        address from;
+        address to;
         uint256 allowance;
         uint256 expiryTime;
-    }
-
-    //Manual blocking allows you to specify a list of blocked address pairs with an associated expiry time for the block
-    struct ManualBlocking {
-        uint256 expiryTime;
+        bytes32 description;
```

Comment:
is `bytes32` enough for the description? we can't even fir an IPFS hash in `bytes32`.

Reply:
I already changed it to string during improving the test cases

Reply:
I think we have to keep the bytes32 as a small description length. Otherwise we can't call batch functions with variable length datatype.
string[], bytes[] are not allowed in 0.4.24;